src/Form/QueueContentsForm.php

```diff
@@ -50,13 +50,15 @@ public function buildForm(array $form, FormStateInterface $form_state) {
 
     $form['mpx_video_types'] = [
       '#type' => 'checkboxes',
-      '#title' => $this->t('mpx Video Types'),
+      '#title' => $this->t('Mpx video types'),
```

Comment:
Certainly odd at first sight, but I left this as lowercase `mpx` on purpose, as per https://github.com/Lullabot/media_mpx#about-the-mpx-name. 

Fine with the change if I got something wrong, 👍

Reply:
Yeah, it is strange at second glance. I was following Drupal convention here which capitalizes the first word of a form label and lower cases the rest. This is a bit of an edge case, since the first word is an acronym that refuses to be all caps 😆 . Maybe the middle ground is `mpx Video types`. There is some precedence for this in the mpx documentation:

<img width="426" alt="mpx_capitalization" src="https://user-images.githubusercontent.com/191049/60809764-6f1f6300-a159-11e9-9083-c1f9422e94de.png">